conanfile.txt

```diff
@@ -0,0 +1,18 @@
+[requires]
+# Core
+spdlog/1.9.2
+glm/0.9.9.5
+
+# Implementation
+opengl/system
+glew/2.2.0
+sdl/2.0.20
```

Comment:
SDL is taking over 15 minutes to build on GitHub. This will probably get annoying with the cache expiring every 10 days of inactivity. Might be a good idea to got back to FetchContent for this one which used the system installed SDL